data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/gold/gold_wfl_no_sides.cfg

```diff
@@ -0,0 +1,129 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [gold]amount=
+##
+# Actions:
+# Give gold to all sides
+# Add gold to all sides
+# Remove gold from all sides
+# Remove enough gold from all sides to have negative gold
+##
+# Expected end state:
+# All sides are affected by the [gold] tag
+#####
+{GENERIC_UNIT_TEST "gold_wfl_no_sides" (
+    [event]
+        name=side 1 turn 1
+        
+        # confirm initial gold amounts
+        [store_gold]
+            side=1
+            variable=initial_gold_side1
+        [/store_gold]
+        [store_gold]
+            side=2
+            variable=initial_gold_side2
+        [/store_gold]
+        
+        {ASSERT (
+            [variable]
+                name=initial_gold_side1
+                numerical_equals=100
+            [/variable]
+        )}
+        
+        {ASSERT (
+            [variable]
+                name=initial_gold_side2
+                numerical_equals=100
+            [/variable]
+        )}
+        
+        # add gold
+        [gold]
+            amount="$($side_number+50)"
```

Comment:
side_number would initially give idea we test that sides get different gold. Would be more fitting with turn_number